From: Birger Schacht <schacht@cert.at>
Date: Thu, 11 Feb 2021 12:38:22 +0100
Subject: Fix core include path

---
 app/webroot/index.php | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/app/webroot/index.php b/app/webroot/index.php
index 59971c6..caba967 100644
--- a/app/webroot/index.php
+++ b/app/webroot/index.php
@@ -68,7 +68,8 @@ if (!defined('APP_DIR')) {
  *
  * Leaving this constant undefined will result in it being defined in Cake/bootstrap.php
  */
-	define('CAKE_CORE_INCLUDE_PATH', ROOT . DS . APP_DIR . DS .'Lib' . DS . 'cakephp' . DS . 'lib');
+	#define('CAKE_CORE_INCLUDE_PATH', ROOT . DS . APP_DIR . DS .'Lib' . DS . 'cakephp' . DS . 'lib');
+	define('CAKE_CORE_INCLUDE_PATH', '/usr/share/misp/Vendor/cakephp/cakephp/lib');
 
 /**
  * Editing below this line should NOT be necessary.
